packages/keyboard-shortcuts/README.md

```diff
@@ -0,0 +1,32 @@
+# Keycodes
```

Comment:
I assume this was meant to be "Keyboard Shortcuts"? 😃 (Guessing it may have been missed for revision in a copy paste)

Reply:
Copy/paste? me? never

Reply:
Fixed in 8c6916c2de

Reply:
Thanks

Reply:
The commit doesn't seem right though :) copy/paste issue?

Reply:
> The commit doesn't seem right though :) copy/paste issue?

🤦‍♂ ac818aed0ae8179dee7d219969ecc656b9dc181e